src/components/ContactDetails.vue

```diff
@@ -874,7 +874,13 @@ section.contact-details {
 		}
 	}
 }
-
+::v-deep .v-select.select {
+	min-width: 0 !important;
```

Comment:
https://github.com/nextcloud/nextcloud-vue/blob/34ec25f79b8e5f5e74aa525339e4efd48e9cc26f/src/components/NcSelect/NcSelect.vue#L989 likely causing issues if the original component assumes a minimum width

Reply:
without `min-width: 0 !important;` the whole form is left aligned and the left part of is under the second column.